src/platform_impl/windows/window.rs

```diff
@@ -475,6 +475,11 @@ impl Window {
         });
     }
 
+    #[inline]
+    pub fn is_minimized(&self) -> Option<bool> {
+        Some(unsafe { IsIconic(self.hwnd()) == 1 })
```

Comment:
Closer to the official docs: `If the window is iconic, the return value is nonzero.`
```suggestion
        Some(unsafe { IsIconic(self.hwnd()) != false.into() })
```

Reply:
I'm not too happy with the additional `unwraps` required due to returning an `Option` as it's not immediately clear that it always will be `Some`. Maybe a `is_minimized :: HWND -> bool` variant in `util` and the function above just returns `Some(util::is_minimized(self.hwnd()))`. Internally we would use the `util.:is_minimized` to avoid the `unwrap` calls

Reply:
yeah sounds better.